sqlite-cloud/quick-start-fastapi-sqlalchemy.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: SQLAlchemy ORM Quick Start Guide
+description: Get started with SQLite Cloud using SQLAlchemy ORM in FastAPI.
+category: getting-started
+status: publish
+slug: quick-start-sqlalchemy-orm
+---
+
+In this quickstart, we will show you how to get started with SQLite Cloud and SQLAlchemy by building a FastAPI backend that connects to and reads from a SQLite Cloud database.
+
+---
+
+1. **Set up a SQLite Cloud account**
+  - If you haven't already, [sign up for a SQLite Cloud account](https://sqlitecloud.io/register) and create a new project.
+  - In this guide, we will use the sample datasets that come pre-loaded with SQLite Cloud.
+
+2. **Create a new Python project**
+  - You should have the latest Python version (3) installed locally.
+
+```bash
+mkdir sqlalchemy-quickstart
+cd sqlalchemy-quickstart
+
+# open the project in VSCode
+# code .
+
+python3 -m venv .venv
+. .venv/bin/activate
+```
+
+3. **Install dependencies**
+  - Run this command from your current directory:
+
+```bash
+pip install "fastapi[standard]" sqlalchemy sqlalchemy-sqlitecloud
+```
+
+  - Do NOT remove the quotes around the FastAPI package.
+  - `sqlalchemy-sqlitecloud` includes `sqlitecloud`, so no need to install the latter separately.
+  - Update `.venv/lib/python{version}/site-packages/sqlitecloud/__init__.py`:
+
+```py
+from .dbapi2 import *
+
+# from .dbapi2 import (
+#     PARSE_COLNAMES,
+#     ...
+# )
+```
```

Comment:
is this a workaround for the `AttributeError: module 'sqlitecloud.dbapi2' has no attribute 'sqlite_version_info'`?

Reply:
Yes. I will remove it now that you've implemented the fix in the `sqlitecloud` package's `__init__.py`.

Reply:
See new Quickstart [PR](https://github.com/sqlitecloud/docs/pull/95).